src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1164,6 +1476,35 @@ inline void rpp_store16_f32_to_u8_avx(Rpp8u *dstPtr, __m256 *p)
     _mm_storeu_si128((__m128i *)dstPtr, px[0]);
 }
 
+inline void rpp_load48_f32pkd3_to_f32pln3_avx512(Rpp32f *srcPtr, __m512 *p)
+{
+    __m512i px[2];
+    px[0] = _mm512_loadu_si512((__m512i *)srcPtr);
```

Comment:
Are you getting correct outputs for F32 cases?
Because the input is float, we should use _mm512_loadu_ps and entire functions need to change w.r.t that